lab3/client/src/views/generator.js

```diff
@@ -0,0 +1,25 @@
+import { Generator } from '../models/generator';
+
+export default function (store) {
+ return class GeneratorComponent extends window.HTMLElement {
+  constructor () {
+   super();
+   this.store = store;
+
+   // TODO: render generator initial view
+   this.generatorValue = this.store.state.generators;
+
+   // TODO: subscribe to store on change event
+   this.onStateChange = this.handleStateChange.bind(this);
```

Comment:
This will throw syntax error because there is no method called `handlerStateChange`